src/docs/sdk/event-payloads/properties/measurements.mdx

```diff
@@ -0,0 +1,18 @@
+`measurements`
+
+: _Optional_. An object containing standard/custom measurements with keys signifying the name of the measurement.
+
+Standard measurement keys currently supported are from the set  `"fp" | "lcp" | "fid" | "ttfb" | "ttfb.requesttime"`.
+
+Custom measurements need units to be specified. Units currently supported are durations from the set  `"ns" | "ms" | "s"`.
```

Comment:
and these I took from [`relay/MetricUnit`](https://github.com/getsentry/relay/blob/1e45a8817e45408ecbdbd3f0a679a5f5e4885290/relay-metrics/src/protocol.rs#L47-L52).

Reply:
Could you check which units would actually make sense and how we would like to specify units? What we have in Relay was only a stub, and now is a good time to rethink this. Generally, we have two options:
- Define a strict allow list of units that we define in Relay, and then propagate down.
- Allow arbitrary strings as unit. Relay still lists well-known units that it can handle, eg. for conversion.

Reply:
ok so making a list of potentially useful categories, maybe product people have to add here.


| type | units | remarks |
| ---- | ---- | ---- |
| time | ns, microseconds, ms, s, m | longer than minute not useful, formatting for microseconds |
| memory | bytes, KiB, MiB, GiB, TiB | KB vs KiB |
| unitless / absolute counts | |
| unitless / percentages | % |

Formatting consistently for some cases is problematic if we go with short forms, for instance `microseconds` vs `ns` and I don't know if we want to get into unicode mess here.

Reply:
I'm not sure of the value provided by arbitrary user strings.
There's no semantic associated with a random `foobar` unit, so I assume it will just be a passthrough all the way to wherever the metric will be displayed on the UI. I think it is more valuable to actually look at valid measurement use cases and we can always extend our supported list with things that make sense.

Reply:
ok looking around a bit, should have done this first
* [Datadog](https://docs.datadoghq.com/metrics/units/)
* [The Unified Code for Units of Measure](https://ucum.org/ucum.html)
* [OpenTelemetry unmerged PR on units spec](https://github.com/open-telemetry/opentelemetry-specification/pull/1177)